trade-finance-manager-ui/server/types/view-models/record-correction/record-correction-log-details-view-model.ts

```diff
@@ -0,0 +1,10 @@
+import { RecordCorrectionLogFields } from '@ukef/dtfs2-common';
+import { BaseViewModel } from '../base-view-model';
+
+export type RecordCorrectionLogDetailsViewModel = BaseViewModel & {
+  mappedCorrectionLog: RecordCorrectionLogFields;
```

Comment:
No mapping actually occurs from the api response to this so this name feels misleading, but also, does this not have additional fields that aren't needed? We don't use `isCompleted` in the template for example do we? I think perhaps don't use `feeRecordId` or `correctionId` either

Reply:
We use isCompleted for the status.  So it does need to be returned to this controller.  Also have removed feeRecordId and correctionId and renamed to correctionDetails